packages/cloudflare/src/api/overrides/incremental-cache/kv-incremental-cache.ts

```diff
@@ -23,69 +23,30 @@ class KVIncrementalCache implements IncrementalCache {
     key: string,
     isFetch?: IsFetch
   ): Promise<WithLastModified<CacheValue<IsFetch>> | null> {
-    const cfEnv = getCloudflareContext().env;
-    const kv = cfEnv.NEXT_INC_CACHE_KV;
-    const assets = cfEnv.ASSETS;
-
-    if (!(kv || assets)) {
-      throw new IgnorableError(`No KVNamespace nor Fetcher`);
-    }
+    const kv = getCloudflareContext().env[BINDING_NAME];
+    if (!kv) throw new IgnorableError("No KV Namespace");
 
-    this.debug(`Get ${key}`);
+    debugCache(`Get ${key}`);
 
     try {
-      let entry: {
-        value?: CacheValue<IsFetch>;
-        lastModified?: number;
-        status?: number;
-      } | null = null;
-
-      if (kv) {
-        this.debug(`- From KV`);
-        const kvKey = this.getKVKey(key, isFetch);
-        entry = await kv.get(kvKey, "json");
-        if (entry?.status === STATUS_DELETED) {
-          return null;
-        }
-      }
+      const entry = await kv.get<IncrementalCacheEntry<IsFetch> | CacheValue<IsFetch>>(
+        this.getKVKey(key, isFetch),
+        "json"
+      );
 
-      if (!entry && assets) {
-        this.debug(`- From Assets`);
-        const url = this.getAssetUrl(key, isFetch);
-        const response = await assets.fetch(url);
-        if (response.ok) {
-          // TODO: consider populating KV with the asset value if faster.
-          // This could be optional as KV writes are $$.
-          // See https://github.com/opennextjs/opennextjs-cloudflare/pull/194#discussion_r1893166026
-          entry = {
-            value: await response.json(),
-            // __BUILD_TIMESTAMP_MS__ is injected by ESBuild.
-            lastModified: (globalThis as { __BUILD_TIMESTAMP_MS__?: number }).__BUILD_TIMESTAMP_MS__,
-          };
-        }
-        if (!kv) {
-          // The cache can not be updated when there is no KV
-          // As we don't want to keep serving stale data for ever,
-          // we pretend the entry is not in cache
-          if (
-            entry?.value &&
-            "kind" in entry.value &&
-            entry.value.kind === "FETCH" &&
-            entry.value.data?.headers?.expires
-          ) {
-            const expiresTime = new Date(entry.value.data.headers.expires).getTime();
-            if (!isNaN(expiresTime) && expiresTime <= Date.now()) {
-              this.debug(`found expired entry (expire time: ${entry.value.data.headers.expires})`);
-              return null;
-            }
-          }
-        }
+      if (!entry || "lastModified" in entry) {
```

Comment:
I think the following might be more readbale:

```ts
if (!entry) {
  return null;
}

return {
  value: entry,
  // blabla
  lastModified: entry.lastModified ?? BUILD_TS
};        
```

Reply:
you'd need a ternary for both value and lastModified due to typescript at which point it isn't really an improvement...

could split that if statement into two though if that helps?

Reply:
> could split that if statement into two though if that helps?

I think it would help understanding the code. But up to you (fine with either)

Reply:
```suggestion
      if (!entry) return null;
      
       if ("lastModified" in entry) {
```